include/nccl_ofi_rdma.h

```diff
@@ -88,8 +88,12 @@ typedef struct nccl_net_ofi_rdma_ctrl_fl_item {
 } nccl_net_ofi_rdma_ctrl_fl_item_t;
 
 /* Structure used to store bounce buffers in a free list */
+/* For LL/LL128 protocols, bounce buffers (source of RDMA read operations) need to be 128B aligned */
```

Comment:
You stuck a bunch of stuff between the comment describing a `nccl_net_ofi_rdma_bounce_fl_item_t` and the definition of the structure.  That's a bit weird.